Frameworks/CoreText/CTTypesetter.mm

```diff
@@ -116,24 +126,39 @@ static CFIndex _DoWrap(CTTypesetterRef ts, CFRange range, WidthFinderFunc widthF
 
         int curChar = chars[curIndex];
 
-        if (curChar != 10 && curChar != 13 && curChar != 9) {
-            characters.push_back((WORD)curChar);
-        } else {
+        // FT does not seem to be able to print tabs, so we have to replace it with a space for now
```

Comment:
No, this will no longer be an issue once we switch away from freetype

---
In reply to: [74674436](https://github.com/Microsoft/WinObjC/pull/873#discussion_r74674436) [](ancestors = 74674436)